File: FireflowSuite/engine/src/org/fireflow/engine/RuntimeContextFactory.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

package org.fireflow.engine;

import org.fireflow.engine.calendar.DefaultCalendarService;
import org.fireflow.engine.condition.ConditionResolver;
import org.fireflow.engine.definition.DefinitionService4DBMS;

/**
 *
 * @author chennieyun
 */
public class RuntimeContextFactory {
    private static RuntimeContext ctx = null;
    
//    public static getRuntimeContext(){
//        if (ctx==null){
//            ctx = RuntimeContext.getInstance();
//            ctx.setCalendarService(new DefaultCalendarService());
//            ctx.setConditionResolver(new ConditionResolver());
//            ctx.setDefinitionService(new DefinitionService4DBMS());
//        }
//    }
}

File: trunk/dot-net_version/ISM.FireWorkflow/trunk/ISM.FireWorkflow/Engine/RuntimeContextFactory.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ISM.FireWorkflow.Engine
{

    /// <summary>
    /// 在没有spring环境下构建RuntimeContext实例。<br/>
    /// (暂未实现)
    /// </summary>
    public class RuntimeContextFactory
    {
        private static RuntimeContext ctx ;

        //    public static getRuntimeContext(){
        //        if (ctx==null){
        //            ctx = RuntimeContext.getInstance();
        //            ctx.setCalendarService(new DefaultCalendarService());
        //            ctx.setConditionResolver(new ConditionResolver());
        //            ctx.setDefinitionService(new DefinitionService4DBMS());
        //        }
        //    }
    }
}
